core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/main/dot-edit-page-main/dot-edit-page-main.component.html

```diff
@@ -1,8 +1,13 @@
 <router-outlet></router-outlet>
 <dot-edit-page-nav
     [pageState]="pageState$ | async"
+    (openOverlayPanel)="showPageTools()"
```

Comment:
This will only emit for the page tools and nothing more... but are you sure is good to emit out or maybe we should encapsulate that inside `dot-edit-page-nav`?